components/Votes/Votes.tsx

```diff
@@ -335,6 +335,11 @@ export function Votes() {
           ))}
         />
       </VotesTableWrapper>
+      {isDirty() ? (
+        <RecommittingVotesMessage>
+          * Changes to committed votes need to be re-committed
+        </RecommittingVotesMessage>
```

Comment:
this was the only change needed afaik to do this, though i do realize the dirty logic isnt implemented the best, its just really fragile

Reply:
I must have misunderstood what the dirty logic does. What exactly does it mean for the input to be dirty? I thought that inputs are dirty if they have any input, not only if they are changing an existing value.

Reply:
you are probably right, but in this case its if the value has changed from a previous commit. Should have commented it